common/lib/mandrill_client/client.py

```diff
@@ -38,40 +38,55 @@ class MandrillClient(object):
     REFERRAL_FOLLOW_UP_EMAIL = 'referred-learner-follow-up'
     REFERRAL_SOCIAL_IMPACT_TOOLKIT = 'social-impact-toolkit'
     MINI_COURSE_ENROLMENT = 'mini-course-enrolment'
+    SEND_ACTION_PLAN = 'send-action-plan'
 
     def __init__(self):
         self.mandrill_client = mandrill.Mandrill(settings.MANDRILL_API_KEY)
 
-    def send_mail(self, template_name, user_email, context, attachments=[], subject=None):
+    def attach_emails(self, user_emails):
+        email_tokens = user_emails.split(',')
+        receiver_emails = []
+        for email in email_tokens:
+            receiver_emails.append({'email': email})
```

Comment:
- We can use list comprehension, Because it's not too complex so it won't affect readability.
```
receiver_emails = [{'email': email} for email in email_tokens]
```
- Also didn't get the token thingy can we please rename it to just emails?

Reply:
Changed